src/main/java/pl/javastart/task/Calculator.java

```diff
@@ -0,0 +1,18 @@
+package pl.javastart.task;
+public class Calculator {
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu